usaspending_api/disaster/v2/views/award/count.py

```diff
@@ -27,7 +27,7 @@ def post(self, request: Request) -> Response:
         count = FinancialAccountsByAwards.objects.filter(*filters)
         if self.award_type_codes:
             count = count.values("award_id").filter(award__type__in=self.filters.get("award_type_codes"))
-        count = self.when_non_zero_award_spending(count.annotate(unique_c=self.unique_file_c)).aggregate(
+        count = self.when_non_zero_award_spending(count.annotate(unique_c=self.unique_file_c_awards)).aggregate(
```

Comment:
this is getting confusing...

At first I though this needs to count file d, then I realized it should match award/amount.

We should just remove this endpoint

Reply:
Preachin' to the choir...